### PR TITLE
[#530] build against latest photon

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
could you have a look at that. am busy until friday

**Comment by cdietrich** [comment]
btw why is org.eclipse.draw2d.feature.group" not on the platofrm update site

**Comment by kthoms** [comment]
could do that
draw2d is considered legacy as of GEF4

File: releng/org.eclipse.xtext.target/org.eclipse.xtext.target-photon.target [hunk 1]
```diff
@@ -66,34 +66,50 @@
...
+		<unit id="org.eclipse.xpand" version="0.0.0"/>
+		<unit id="org.eclipse.xtend" version="0.0.0"/>
+		<unit id="org.eclipse.xtend.typesystem.emf" version="0.0.0"/>
+		<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/releases/xpand/R201406030414"/>
+	</location>
+	
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit/"/>
-		<unit id="org.junit" version="0.0.0"/>
+		<repository location="http://download.eclipse.org/tools/orbit/downloads/drops/S20180119201206/repository"/>
```
Comment (cdietrich) [inline releng/org.eclipse.xtext.target/org.eclipse.xtext.target-photon.target]: please sort to the end

**Comment by cdietrich** [comment]
looks good now. please merge

**Pull request merged**